sample/src/main/java/com/vimeo/sample/model/JsonAdapterExample.java

```diff
@@ -0,0 +1,37 @@
+package com.vimeo.sample.model;
+
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import java.util.Date;
+
+@UseStag
+public class JsonAdapterExample {
```

Comment:
Could we make one of the fields in this class `NonNull` so that we have a test case for the null check?